docs/resources/stream_dashboard.md

```diff
@@ -0,0 +1,2 @@
+---
+Stream Dashboards are deprecated and will be removed shortly. DO NOT USE.
```

Comment:
I assume this is true and why the `stream_dashboard.md` file was missing before this PR.

Reply:
yes, that's the case.